_layouts/post.html

```diff
@@ -158,7 +158,7 @@ <h5>
       "dateModified": "{{page.updated_at}}",
       "author": {
         "@type": "Organization",
```

Comment:
If there is an author we can toggle to `"@type": "Person"` otherwise default to `"Organization"`